### PR TITLE
chore: [SIW-1479] Moved io-web jwt validation to APIM

### PR DESCRIPTION
**Pull request by silvicir**
#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by grausof — APPROVED** [review]
Great cleaning. I haven't tested but I trust it 👏🏻

File: infra/resources/_modules/apim/api/ioweb/_base_policy.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            <openid-config url="https://wallet.io.pagopa.it/exchange/openid-configuration" />
+            <required-claims>
+                <claim name="fiscal_number" />
+            </required-claims>
+        </validate-jwt>
+        <!-- get the claim fiscal_number -->
+        <set-variable name="fiscalCode" value="@((string)((Jwt)context.Variables["jwt"]).Claims.GetValueOrDefault("fiscal_number"))" />
+        <!-- send the request to trial system -->
+        <send-request mode="new" response-variable-name="trialSystemResponse" timeout="5">
+            <set-url>@($"https://api.trial.pagopa.it/manage/api/v1/trials/{{trial-id}}/subscriptions/{context.Variables["fiscalCode"]}")</set-url>
```
Comment (gunzip) [inline infra/resources/_modules/apim/api/ioweb/_base_policy.xml:30]: what if this call fails? myabe we should return some explicit error to the caller ie. "Error retrieving activation status"

File: infra/resources/_modules/apim/api/ioweb/_get_current_wallet_instance_status_policy.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <inbound>
+        <base />
+        <!-- forwards the call to the Function App -->
+        <set-header name="Authorization" exists-action="delete" />
+        <set-method>POST</set-method>
+        <set-header name="Content-Type" exists-action="override">
+            <value>application/json</value>
+        </set-header>
+        <set-body>@{
+            var fiscalCode = context.Variables["fiscalCode"];
```
Comment (gunzip) [inline infra/resources/_modules/apim/api/ioweb/_get_current_wallet_instance_status_policy.xml:11]: what if it's not set?

File: infra/resources/_modules/apim/api/ioweb/_set_wallet_instance_status_policy.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <base />
+        <!-- forwards the call to the Function App -->
+        <set-header name="Authorization" exists-action="delete" />
+        <set-method>PUT</set-method>
+        <set-header name="Content-Type" exists-action="override">
+            <value>application/json</value>
+        </set-header>
+        <set-variable name="requestBody" value="@{ return context.Request.Body.As<string>(preserveContent: true); }" />
+        <set-body>@{
+            var fiscalCode = context.Variables["fiscalCode"];
```
Comment (gunzip) [inline infra/resources/_modules/apim/api/ioweb/_set_wallet_instance_status_policy.xml:12]: ditto

File: infra/resources/_modules/apim/api/ioweb/_set_wallet_instance_status_policy.xml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            var status = context.Variables["requestBody"];
```
Comment (gunzip) [inline infra/resources/_modules/apim/api/ioweb/_set_wallet_instance_status_policy.xml:13]: ditto

File: infra/resources/_modules/apim/api/ioweb/_base_policy.xml (hunk 1 above)
Comment (silvicir) [reply infra/resources/_modules/apim/api/ioweb/_base_policy.xml:31]: Added a TODO comment

**Review by Krusty93 — APPROVED** [review]

**Pull request merged**